src/table/index.js

```diff
@@ -68,17 +68,23 @@ class TableField extends Component {
     this.handleDeletedRow = this.handleDeletedRow.bind(this);
     this.handleRowSelect = this.handleRowSelect.bind(this);
     this.handleAllRowSelect = this.handleAllRowSelect.bind(this);
+    this.checkMyRow = this.checkMyRow.bind(this);
```

Comment:
checkMyRow? Does my stand for anything here? A better name would definitely help.

Reply:
Similar issue in the next line too.

Reply:
updated function name to **isRowIsExpandable**

Reply:
How about isRowExpandable?